tests/test_plugin_features.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pytest
+
+import dclab
+from dclab.rtdc_dataset.plugins import PlugInFeature
+
+from helper_methods import retrieve_data
+
+
+@pytest.fixture
+def cleanup_plugin_features():
+    """Fixture used to setup and cleanup some fake ancillary features"""
+    # code run before the test
+    pass
+    # then the test is run
+    yield
+    # code run after the test
+    # remove our test plugin examples
+    pass
+    # remove_plugins()
+
+
+def compute_single_plugin_feature(rtdc_ds):
+    circ_per_area = rtdc_ds["circ"] / rtdc_ds["area_um"]
+    return circ_per_area
+
+
+def compute_multiple_plugin_features(rtdc_ds):
+    circ_per_area = rtdc_ds["circ"] / rtdc_ds["area_um"]
+    bright_per_area = rtdc_ds["bright_avg"] / rtdc_ds["area_um"]
+    return {"circ_per_area": circ_per_area, "bright_per_area": bright_per_area}
+
+
+def test_single_plugin_feature():
+    ds = dclab.new_dataset(retrieve_data("rtdc_data_hdf5_rtfdc.zip"))
+
+    feature_info = {
+        "feature_name": "circ_per_area",
+        "method": compute_single_plugin_feature,
+        "req_config": [],
+        "req_features": ["circ", "area_um"],
+        "priority": 1,
+    }
+
+    pf = PlugInFeature(**feature_info)
+    assert pf.plugin_registered
+    assert pf.plugin_info == feature_info
+    circ_per_area = ds["circ_per_area"]
+    assert np.allclose(circ_per_area, ds["circ"] / ds["area_um"])
+
+    assert "circ_per_area" in ds
```

Comment:
This assertion should come before line 48, otherwise it is redundant.

Reply:
Nice thanks